docs/modeling/example-fitting-constraints.rst

```diff
@@ -1,44 +1,60 @@
 Fitting with constraints
 ========================
 
-Fitters support constrained fitting.
+`~astropy.modeling.fitting` support constraints, however, different fitters support
+different types of constraints. The `~astropy.modeling.fitting.Fitter.supported_constraints`
+attribute shows the type of constraints supported by a specific fitter::
 
-- All fitters support fixed (frozen) parameters through the ``fixed`` argument
-  to models or setting the `~astropy.modeling.Parameter.fixed`
-  attribute directly on a parameter.
+    >>> from astropy.modeling import fitting
+    >>> fitting.LinearLSQFitter.supported_constraints
+    ['fixed']
+    >>> fitting.LevMarLSQFitter.supported_constraints
+    ['fixed', 'tied', 'bounds']
+    >>> fitting.SLSQPLSQFitter.supported_constraints
+    ['bounds', 'eqcons', 'ineqcons', 'fixed', 'tied']
+
+Fixed Parameter Constraint
+--------------------------
 
-  For linear fitters, freezing a polynomial coefficient means that the
-  corresponding term will be subtracted from the data before fitting a
-  polynomial without that term to the result. For example, fixing ``c0`` in a
-  polynomial model will fit a polynomial with the zero-th order term missing
-  to the data minus that constant. However, the fixed coefficient value is
-  restored when evaluating the model, to fit the original data values::
+All fitters support fixed (frozen) parameters through the ``fixed`` argument
+to models or setting the `~astropy.modeling.Parameter.fixed`
+attribute directly on a parameter.
+
+For linear fitters, freezing a polynomial coefficient means that the
+corresponding term will be subtracted from the data before fitting a
+polynomial without that term to the result. For example, fixing ``c0`` in a
+polynomial model will fit a polynomial with the zero-th order term missing
+to the data minus that constant. However, the fixed coefficient value is
+restored when evaluating the model, to fit the original data values::
```

Comment:
Would it be more accurate to say that "the fixed coefficient(s)  and corresponding term(s) is (are) restored to the fit polynomial and that this is the polynomial returned from the fitter."? The text above suggests that this only happens when the model is evaluated, which seems misleading, and it is perhaps not necessary to add the "to fit the original data values" though perhaps that could be reworded to "to match the original data values"